[ADD] Email marketing: Analyze metrics backport (#10636)
odoo/documentation: content/applications/marketing/email_marketing/analyze_metrics.rst

===============
Analyze metrics
===============

.. |CTR| replace:: :abbr:`CTR (Click through rate)`

To properly understand the success or failure of an email marketing campaign, it is
necessary to monitor several key metrics. The insights gained from these metrics can then be used to
optimize future campaigns. Odoo's **Email Marketing** application tracks several :ref:`key metrics
<email-marketing/key-metrics>`, that can be interpreted to improve future campaigns.

View metrics
============

After a mass mail has been sent, the results for that particular mailing are displayed in multiple
locations.

To access the metrics for an individual mailing, navigate to :menuselection:`Email Marketing app
--> Mailings`. Locate the specific mailing in the list view, and use the column headings to view the
results for that mailing. Click on one of the mailings in the list to open the record. At the top of
the record, detailed metrics are displayed as smart buttons.

.. image:: analyze_metrics/metric-smart-buttons.png
   :align: center
   :alt: The smart buttons on a mass mailing, displaying the results of the message.

.. tip::

>    Click on the smart buttons to see the recipient records for each metric.

Reviewer: ```suggestion
   Click on any smart button to see the recipient records for that respective metric.
```

.. _email-marketing/key-metrics:

> 

Reviewer: I noticed that not all of the smart buttons available on the Mailing form are mentioned, such as 'Opportunities', 'Quotations', 'Invoiced', 'Sent', and 'Ignored'... which is okay, if you are purely trying to mention percentage-based metric buttons -- but, if that's the case, I'd, at the very least, allude to them in one way or another -- that way the reader won't feel like they're "missing" something as they follow along. Just something to consider during your revisions.

Opened rate
-----------

The *Opened* rate measures the percentage of emails opened by recipients against the total number of

> sent emails.

Reviewer: I totally understand the use of italics here, as this is, essentially, a definition -- which is totally fine. You can also consider to go the GUI route, and phrase it like: 
"The :guilabel:`Opened` smart button displays the percentage of ..."

But, honestly, either route is totally fine. Just figured it was worth mentioning for your consideration during your revision. 👍

> A high open rate may indicate that the subject line was timely, compelling, and successfully

Reviewer: ```suggestion
A high open rate may indicate the subject line was timely, compelling, and successfully
```

prompted the recipients to view the message.

A low open rate may indicate the subject line failed to capture the recipients' interest. It could

> also indicate the email ended up in a spam or junk folder due to having a poor sender reputation or

Reviewer: ```suggestion
also indicate the email ended up in a spam or junk folder, due to having a poor sender reputation, or
```
added comma for enhanced sentence pacing/flow

failing to :doc:`configure the proper DNS records <../../general/email_communication/email_domain>`.

Replied rate
------------

The *Replied* rate measures the percentage of recipients who responded to the email directly against
the total number of sent emails.

A high replied rate may indicate the email resonated with recipients, prompting them to take action
or provide feedback.

> A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
> action.

Reviewer: ```suggestion
A low replied rate may suggest that the message lacked relevance, or did not contain a clear
call-to-action.
```

Clicked rate
------------

> The *Clicked* through rate (CTR), measures the percentage of recipients who clicked on a link within

Reviewer: This may be an instance wherein the GUI label of the button might be more useful.

For instance, this could be reworded to:
"The :guilabel:`Clicked` smart button represents the *Click Through Rate (CTR)*, which measures the percentage..."

Or something like that...
```suggestion
The *Clicked* rate, represents the *Click Through Rate (CTR)*, which measures the percentage of recipients who clicked on a link within
```

the email against the total number of sent emails.

A high |CTR| may indicate the email content was relevant and appropriately targeted. Recipients were
motivated to click the links provided, and likely found the content engaging.

> A low |CTR| may indicate issues with either the targeting, or the content itself. If there were any calls to action, recipients may have been unmotivated, or the message itself may have been directed toward the wrong audience.

Reviewer: ```suggestion
A low |CTR| may indicate issues with either the targeting, or the content itself. If there were any
calls-to-action, recipients may have been unmotivated, or the message may have been directed
toward the wrong audience.
```

Received rate
-------------

The *Received* rate measures the percentage of emails successfully delivered against the
total number of sent emails.

> A high received rate can indicate that the mailing list used is up-to-date and the sender

Reviewer: ```suggestion
A high received rate can indicate the mailing list used is up-to-date, and the sender
```

authentication is trusted by email providers.

> A low received rate may indicate issues either with the mailing list used for the mailing or with

Reviewer: ```suggestion
A low received rate may indicate issues with either the mailing list used for the mailing, or with
```

the sender authentication. View the :ref:`email-marketing/deliverability-issues` section for more
information.

Bounced rate
------------

The *Bounced* rate measures the percentage of emails unsuccessfully delivered, against the
total number of sent emails.

> A high bounce rate could indicate issues either with the mailing list used for the mailing or with

Reviewer: ```suggestion
A high bounce rate could indicate issues with either the mailing list used for the mailing, or with
```

the sender authentication.

> A low bounce rate may indicate that the mailing list used is up-to-date and the sender

Reviewer: ```suggestion
A low bounce rate may indicate the mailing list used is up-to-date, and the sender
```

authentication is trusted by email providers. View the :ref:`email-marketing/deliverability-issues`
section for more information.

.. _email-marketing/deliverability-issues:

Deliverability issues
=====================

> The following define possible reasons for a high bounce rate or low received rate:

Reviewer: ```suggestion
The following defines possible reasons for a high bounce rate or low received rate:
```

> - Using a mailing list that contains outdated contact information or malformed email addresses is

Reviewer: ```suggestion
- Using a mailing list that contains outdated contact information, or malformed email addresses, is
```

  likely to result in a high bounce rate and/or a low received rate.
- Mailings sent using a *From* email address that differs from the sender's domain are likely to

>   bounce with certain email providers due to failing :doc:`DMARC authentication

Reviewer: ```suggestion
  bounce with certain email providers, due to failing :doc:`DMARC authentication
```

===============
Analyze metrics
===============

.. |CTR| replace:: :abbr:`CTR (Click through rate)`

> In order to properly understand the success or failure of an email marketing campaign, it is

Reviewer 2: ```suggestion
To properly understand the success or failure of an email marketing campaign, it is
```

necessary to monitor several key metrics. The insights gained from these metrics can then be used to

> optimize future campaigns. Odoo's *Email Marketing* application tracks several :ref:`key metrics

Reviewer 2: ```suggestion
optimize future campaigns. Odoo's **Email Marketing** application tracks several :ref:`key metrics
```

A high open rate may indicate that the subject line was timely, compelling, and successfully
prompted the recipients to view the message.

A low open rate may indicate the subject line failed to capture the recipients' interest. It could
also indicate the email ended up in a spam or junk folder due to having a poor sender reputation or
failing to :doc:`configure the proper DNS records <../../general/email_communication/email_domain>`.

Replied rate
------------

The *Replied* rate measures the percentage of recipients who responded to the email directly against
the total number of sent emails.

A high replied rate may indicate the email resonated with recipients, prompting them to take action
or provide feedback.

A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
action.

Clicked rate
------------

The *Clicked* through rate (CTR), measures the percentage of recipients who clicked on a link within
the email against the total number of sent emails.

A high |CTR| may indicate the email content was relevant and appropriately targeted. Recipients were
motivated to click the links provided, and likely found the content engaging.

> A low |CTR| may indicate issues with either the targeting, or the content itself. Recipients may
> have been unmotivated by the calls to action, if there were any, or the message itself may have been
> directed toward the wrong audience.

Reviewer 2: ```suggestion
A low |CTR| may indicate issues with either the targeting, or the content itself. If there were any calls to action, recipients may have been unmotivated, or the message itself may have been directed toward the wrong audience.
```

Received rate
-------------

> The *Received* rate measures the percentage of emails that were successfully delivered against the

Reviewer 2: ```suggestion
The *Received* rate measures the percentage of emails successfully delivered against the
```

total number of sent emails.

A high received rate can indicate that the mailing list used is up-to-date and the sender
authentication is trusted by email providers.

A low received rate may indicate issues either with the mailing list used for the mailing or with
the sender authentication. View the :ref:`email-marketing/deliverability-issues` section for more
information.

Bounced rate
------------

> The *Bounced* rate measures the percentage of emails that were unsuccessfully delivered, against the

Reviewer 2: ```suggestion
The *Bounced* rate measures the percentage of emails unsuccessfully delivered, against the
```

total number of sent emails.

A high bounce rate could indicate issues either with the mailing list used for the mailing or with
the sender authentication.

A low bounce rate may indicate that the mailing list used is up-to-date and the sender
authentication is trusted by email providers. View the :ref:`email-marketing/deliverability-issues`
section for more information.

.. _email-marketing/deliverability-issues:

Deliverability issues
=====================

The following define possible reasons for a high bounce rate or low received rate:

> - Using a mailing list that contains outdated contact information, or malformed email addresses are

Reviewer 2: ```suggestion
- Using a mailing list that contains outdated contact information or malformed email addresses is
```
